concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -107,8 +107,8 @@ resources:
   - name: graphite-nozzle
     type: git
     source:
-      uri: https://github.com/alphagov/graphite-nozzle
-      branch: 106191208-deaggregated_names
+      uri: https://github.com/alphagov/paas-graphite-nozzle
+      branch: upgrade_go
```

Comment:
Maybe, but whatever we do on this repo, we need to ensure that this manifest is pinned to a specific version. I don't think we want to be in a position where commits to that repo cause an upgrade here without us having a PR here first.